lib/Core/Site/DomainObjectMapper.php

```diff
@@ -190,6 +223,54 @@ public function mapField(APIField $apiField, SiteContent $content)
         ]);
     }
 
+    public function getNullField($identifier, SiteContent $content)
+    {
+        $message = 'nešto je trulo u državi danskoj';
```

Comment:
Quite Shakespearean and fancy, but maybe put some proper message :D

Reply:
Fixed in https://github.com/netgen/ezplatform-site-api/pull/94/commits/42632bc050a2912fb50e058f166702de74a958f3 :)